app/javascript/components/molecules/MultiSelect.js

```diff
@@ -2,21 +2,48 @@ import React from 'react'
 import Select from 'react-select'
 import PropTypes from 'prop-types'
 
+const HEIGHT_NORMAL = 'normal'
+const HEIGHT_SMALL = 'small'
+
+const getStyleOverrides = (controlHeight) => {
+  let controlHeightPx
+  switch (controlHeight) {
+    case HEIGHT_SMALL:
```

Comment:
do we have a case where we have inputs with a height of 38px?

Reply:
Only the inputs inside of the application table.

I think  this is good to keep not because the 38px version will be used, but because it'll be very easy to create inputs of different heights when we do need them

Reply:
ok sounds good